TESTING.md

```diff
@@ -0,0 +1,34 @@
+# Running Tests
+
+## Prerequisites
+Before running tests, ensure all steps in the [BUILDING.md](https://github.com/apache/geode/blob/develop/BUILDING.md) have been performed.
+
+## Running Tests By Type
+Tests are broken up into three types - unit, integration, and distributed.  They can be executed using the following commands from the Geode repository root directory:
+* Unit tests
+  `./gradlew test --parallel`
+* Integration tests
+  `./gradlew integrationTest --parallel`
```

Comment:
You don't want to run integration or distributed tests in parallel without enabling the docker containers.

Reply:
No, its kind of a pain and you need a pretty beefy machine.